backend/src/main/java/com/github/devbk84/backend/service/ProductService.java

```diff
@@ -22,4 +25,30 @@ public List<Product> getAllProducts() {
 
     public Product getProductById(String id) {
         return productRepo.findById(id).orElseThrow(); }
+
+
+    public Product updateEntry(String id, ProductDTO entryUpdate) {
+        Product toEdit = new Product(
+                id,
+                entryUpdate.name(),
+                entryUpdate.description(),
+                entryUpdate.orderFavorites(),
+                entryUpdate.price());
+
+            return productRepo.save(toEdit);
+    }
+
+    public void deleteEntryByID(String id) { productRepo.deleteById(id);
```

Comment:
Hier in der Datei und dann am besten auch überall einmal den Formater anschmeißen mit cmd+option+l